_docs_integrate/data-model-overview.md

```diff
@@ -415,9 +415,9 @@ A Request allows you to ask another Identity to do something. What this "somethi
 
 ### RequestItems
 
-RequestItems can be sent inside of a [Request](#request) and specify what should be done when the [Request is accepted]({% link _docs_use-cases/use-case-consumption-accept-incoming-request.md %}). More information on how to use the individual [types of RequestItems]({% link _docs_integrate/request-and-response-introduction.md %}#types-of-requestitems) for your purposes can be found in the [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}).
+RequestItems can be sent inside of a [Request](#request) and specify what should be done when the [Request is accepted]({% link _docs_use-cases/use-case-consumption-accept-incoming-request.md %}). More information on how to use the individual [types of RequestItems]({% link _docs_integrate/request-and-response-introduction.md %}#types-of-requestitems) for your purposes can be found in the [Request and Response introduction]({% link _docs_integrate/request-and-response-introduction.md %}). If you are interested in a short overview of the various operations which can be performed with Attributes, [Attribute management options]({% link _docs_integrate/attribute-introduction.md %}#attribute-management-options) should also be helpful.
 
-#### AuthenticationRequestItem
+#### [AuthenticationRequestItem]({% link _docs_integrate/request-and-response-introduction.md %}#authenticationrequestitem)
```

Comment:
Personally, I don't like the style of links being used in headlines. If you'd like to have the connection to the Request and Response introduction, I'd suggest to add a sentence at the end of the text below Request here in the data model.

Reply:
The sentence already exists in the beginning of the chapter RequestItem but in the ticket was written: 

_Suggestion: Add a one-sentence description of the purpose of every request item between its headline and its table of fields. Alternatively, feel free to attribute the confusion to stupidity and close this ticket._

So instead of repeating I thought to add a direct link there...

Reply:
I see. So we already provide the link, but it's not prominent enough for people to click it. I think the same problem might occur if we just have the headlines as link. Thus, I'd suggest you indeed add short sentences between the headlines and the tables of the RequestItems. But instead of copying the texts from the Request and Response introduction, you write that for more information they should checkout the corresponding section of the Request and Response introduction and add the links to sections that you currently have in the headline. For those RequestItems that have another dedicated guide (Share Attributes with peer, Read Attributes from peer, Request Consent from peer...) you can also mention and link those pages. I hope this will be fool-proof then. :)